modules/standard/BigInteger.chpl

```diff
@@ -2517,15 +2517,15 @@ When ``numer/denom`` does not produce an integer, this method may produce incorr
   /*
     .. warning::
 
-       bigint.divisible_p is deprecated, use bigint.isDivisible instead
+       bigint.divisible_p is deprecated, use bigint.isDivisibleByPowerOf2 instead
```

Comment:
Oh oops, looks like this documentation warning wasn't updated to say `divisible_2exp_p`.  Sorry for missing that!

Reply:
Oh sorry, don't know how I missed that!
I have corrected it now..

Reply:
It's easy to do XD